components/Transactions/CoinPicker.tsx

```diff
@@ -44,4 +52,19 @@ const styles = StyleSheet.create({
   pickerItem: {
     color: "black",
   },
+  Box:{
+    width: 215,
```

Comment:
Did you test this on another phone? Does it look good on different screen sizes?

Reply:
yes, it looks like your phone